<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>831</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="MyOpenGLWidget" name="openGLWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,15">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_hide_show">
            <property name="minimumSize">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_control" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="bottomMargin">
              <number>7</number>
             </property>
             <item>
              <widget class="QLabel" name="label_color">
               <property name="minimumSize">
                <size>
                 <width>187</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>颜色</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_colors"/>
             </item>
             <item>
              <widget class="QLabel" name="label_line">
               <property name="minimumSize">
                <size>
                 <width>187</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>分隔线</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QCheckBox" name="checkBox_row">
                 <property name="text">
                  <string>横向分隔线</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="checkBox_col">
                 <property name="text">
                  <string>纵向分隔线</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_display">
                 <property name="text">
                  <string>是否显示</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,5">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>像素间隔:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_margin">
                   <property name="minimumSize">
                    <size>
                     <width>33</width>
                     <height>21</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_function">
               <property name="minimumSize">
                <size>
                 <width>187</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>功能</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton_import_txt">
                 <property name="text">
                  <string>导入txt文件</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButton_export_txt">
                 <property name="text">
                  <string>导出为txt文件</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButton_import_pic">
                 <property name="text">
                  <string>导入图片</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton_export_pic">
                 <property name="text">
                  <string>导出为图片</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_pixelSetting"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_pixelSetting">
   <property name="text">
    <string>像素点数量</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
